packages/wrangler/src/d1/index.ts

```diff
@@ -70,6 +71,28 @@ export const d1 = (yargs: Argv) => {
 				Execute.Options,
 				Execute.Handler
 			)
+			.command("migrations", "Interact with D1 Migrations", (yargs2) =>
+				yargs2
+					.command(
+						"list <name>",
+						"List your D1 migrations",
+						Migrations.ListOptions,
+						Migrations.ListHandler
+					)
+					.command(
+						"create <name> <message>",
+						"Create a new Migration",
+						Migrations.CreateOptions,
+						Migrations.CreateHandler
+					)
+					.command(
+						"apply <name>",
```

Comment:
I feel like name is confusing here. It made me check the code to see if this only applied one migration. We should really have as useful naming as possible

```suggestion
						"list <database>",
						"List your D1 migrations",
						Migrations.ListOptions,
						Migrations.ListHandler
					)
					.command(
						"create <database> <message>",
						"Create a new Migration",
						Migrations.CreateOptions,
						Migrations.CreateHandler
					)
					.command(
						"apply <database>",
```

Reply:
Agreed, fixed